.clang-format

```diff
@@ -1,184 +1,203 @@
 ---
+DisableFormat:   false
+
 Language:        Cpp
-# BasedOnStyle:  Mozilla
-AccessModifierOffset: -4
-AlignAfterOpenBracket: Align
+Standard:        c++20
+
+ColumnLimit:     120
+UseCRLF:         false
+
+#UseTab:          AlignWithSpaces
+UseTab:          Never
+TabWidth:        4
+IndentWidth:     4
+ConstructorInitializerIndentWidth: 4
+
+IndentPPDirectives: AfterHash
+PPIndentWidth:   1
```

Comment:
Based on the example in the clang-format docs, I think we need to bump this to at least `2`. One space for indentation can make it difficult to follow the logic.